srcs/runners_manager/vm_creation/gcloud/GcloudManager.py

```diff
@@ -73,6 +74,16 @@ def configure_instance(
         )
         disk_size_gb = runner.vm_type.config["disk_size_gb"]
         disk_type = f"projects/{self.project_id}/zones/{self.zone}/diskTypes/pd-ssd"
+
+        automatic_restart = True
```

Comment:
I'm not sure if VMs are currently created with `automatic_restart=true`. If that's not the case, it's better to set it to `false` in all situations.

Reply:
I don't think they are, but might be interesting to play with this value for instances that are idle and scheduled with SPOT.

Reply:
It's impossible to use `automatic_restart=true` for a SPOT instance. But in the other scenario I think it's a good idea to have it !